drafts/draft-ietf-ippm-ioam-data-integrity.xml

```diff
@@ -786,6 +786,9 @@
           then the transit node MUST NOT generate a signature and MUST NOT
           update the Signature field.</t>
 
+          <t>For an Integrity Protected DEX Option-Type, each transit node acts
```

Comment:
Given that in case of DEX only header fields would be integrity protected, IMHO it is sufficient that *at least* one node acts as as the validator. This could be the decapsulating node or any other node, but I'm not sure why we would mandate that *every* transit node acts as a validator. Could you elaborate?

Reply:
There are probably two positions here:
* You don't want a node to act on the instructions in the IOAM header if the packet has been compromised. Then you indeed want every node to act as validator. But this isn't only true for DEX but also for tracing. I.e., we need header validation at every hop.
* You're ok with a basic integrity check of the header at the end. In that case, you might face the situation that a transit node changed the header and other nodes acted on the wrong header. You discover that at the decap node - but the damage was already done earlier.